contracts/AssetToken.sol

```diff
@@ -2,16 +2,16 @@
 
 // SPDX-License-Identifier: LGPL-3.0+
 
-pragma solidity 0.5.0;
+pragma solidity ^0.5.0;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
-import "./ERC20Interface.sol";
+import "../openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
```

Comment:
This works around a problem with solidity-coverage. node_modules doesn't get copied to the coverageEnv, so can not instrument those files